include/multipass/standard_paths.h

```diff
@@ -28,32 +28,13 @@ namespace multipass
 class StandardPaths : public Singleton<StandardPaths>
 {
 public:
-    // TODO try to replace all the stuff below with `using enum` in C++20 (P1099R5)
     using LocateOption = QStandardPaths::LocateOption;
-    static constexpr auto LocateFile = LocateOption::LocateFile;
-    static constexpr auto LocateDirectory = LocateOption::LocateDirectory;
+    using enum QStandardPaths::LocateOption;
```

Comment:
The `QStandardPaths::` bit here might seem redundant, but MSVC didn't accept this without it. Probably a bug in MSVC, but even with the redundant text, it's still a big improvement.